edx_sga/settings.py

```diff
@@ -0,0 +1,102 @@
+"""
```

Comment:
Can you rename this file to `test_settings.py` to make it clear that these settings are never meant for use in production?